### ISSUE TITLE
Validation function enforces default of SGE scheduler at cluster creation time

### ISSUE THREAD

**Issue by kotfic** [Kitware/cumulus#164]
@cjh1 
See [here](https://github.com/Kitware/cumulus/blob/master/girder/cumulus/server/models/cluster.py#L68-L75). For ansible clusters scheduler type should be determined at provision time.

**Comment by cjh1** [comment]
Yes, we currently always default to SGE as that is what StarCluster provides
